xbmc/games/dialogs/osd/DialogGameVideoFilter.cpp

```diff
@@ -250,19 +285,101 @@ void CDialogGameVideoFilter::PostExit()
 
 bool CDialogGameVideoFilter::OnClickAction()
 {
+  if (static_cast<int>(m_focusedItemIndex) < m_items.Size())
+  {
+    const std::shared_ptr<CFileItem> focusedItem = m_items[m_focusedItemIndex];
```

Comment:
I think this could even be `const std::shared_ptr<const CFileItem>`.

Reply:
It can't be here, because it needs a std::const_pointer_cast, but I actually like the idea of a const cast, so I'll do that.